mvg/plotting.py

```diff
@@ -248,27 +258,35 @@ def _plot_row(row_data, is_uncert_data, y_pos=0):
             rect = patches.Rectangle(
                 (width * start_pos, y_pos),
                 width * block_len,
-                height - y_pos,
+                row_height,
                 edgecolor=colors[col],
                 facecolor=colors[col],
                 fill=True,
             )
             axes.add_patch(rect)
 
     datalist = data["labels"].tolist()
-    _plot_row(datalist, is_uncert_data=False)
+    _plot_row(
+        datalist,
+        is_uncert_data=False,
+        y_pos=row_pos_height[0][0],
+        row_height=row_pos_height[0][1],
+    )
 
     if show_uncertain:
         uncertlist = data["uncertain"].tolist()
-        _plot_row(uncertlist, is_uncert_data=True, y_pos=4 / 5 * height)
+        _plot_row(
+            uncertlist,
+            is_uncert_data=True,
+            y_pos=row_pos_height[1][0],
+            row_height=row_pos_height[1][1],
+        )
 
     # Create time ticks on x-axis and labels
     if timeticks_interval is None:
-        tick_index = (
-            [0]
-            + [i for i in range(1, len(datalist)) if datalist[i] != datalist[i - 1]]
-            + [len(datalist) - 1]
-        )
+        tick_index = [0] + [
+            i for i in range(1, len(datalist)) if datalist[i] != datalist[i - 1]
+        ]
```

Comment:
same issue here with missing the last index for plotting

Reply:
This was done because, since the last timestamp (the fake one) has a label None, we want to skip adding a tick for it.